salt/grains/core.py

```diff
@@ -1482,7 +1482,7 @@ def os_data():
     grains.update(_ps(grains))
 
     # Load additional OS family grains
-    if grains['os_family'] == "RedHat":
+    if grains['os_family'] in ['RedHat', 'SUSE']:
         grains['osmajorrelease'] = grains['osrelease'].split('.', 1)[0]
```

Comment:
Dunno about RH. We provide standard `osrelease_info` as an array of two elements: [major, minor]. I wound reuse that, instead of spaghettify...